src/lib/support/logging/CHIPLogging.h

```diff
@@ -327,11 +327,17 @@ static inline void GetModuleName(char * buf, uint8_t module)
 
 extern uint8_t gLogFilter;
 
-#define IsCategoryEnabled(CAT) ((CAT) <= gLogFilter)
+inline bool IsCategoryEnabled(uint8_t CAT)
```

Comment:
The other inlines in this file have a static qualifier. Is there any reason these do not?